catalog.yml

```
@@ -153,3 +153,7 @@
 - repo_url: https://github.com/greenelab/annorxiver_manuscript
   html_url: https://greenelab.github.io/annorxiver_manuscript/
   preprint_citation: doi:10.1101/2021.03.04.433874
+- repo_url: https://github.com/lotusnprod/rootstock
+  html_url: https://lotusnprod.github.io/rootstock/v/freeze
```

I don't think our catalog software will be able to follow this redirect. For consistency, let's do:

```suggestion
  html_url: https://lotusnprod.github.io/rootstock/
```

---

Fine for me